metadata-ingestion/src/datahub/ingestion/source/powerbi_report_server/report_server.py

```diff
@@ -108,10 +108,21 @@ def host(self):
 
 
 class PowerBiReportServerDashboardSourceConfig(PowerBiReportServerAPIConfig):
-    platform_name: str = "powerbi"
-    platform_urn: str = builder.make_data_platform_urn(platform=platform_name)
-    report_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
-    chart_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
+    platform_name: str = pydantic.Field(
+        default=Constant.PLATFORM_NAME, hidden_from_docs=True
+    )
+    platform_urn: str = pydantic.Field(
+        default=builder.make_data_platform_urn(platform=Constant.PLATFORM_NAME),
+        hidden_from_docs=True,
+    )
+    report_pattern: AllowDenyPattern = pydantic.Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns to filter PowerBI Reports in ingestion.",
+    )
+    chart_pattern: AllowDenyPattern = pydantic.Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Regex patterns to filter PowerBI Charts in ingestion.",
+    )
```

Comment:
If I'm not wrong, these two patterns are not used anywhere and so the feature actually requires further work.

Whether they were added as a sort of TODO or not, I don't know 🤔

Reply:
I suspect they were added as a form of TODO. Added a check for the report_pattern, chart_pattern and platform_urn however are not used so deleted those.